packages/search/addon/services/navi-report-search-provider.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * This service is used to search for reports stored in the persistence layer.
+ */
+
+import { inject as service } from '@ember/service';
+import NaviBaseSearchProviderService from './navi-base-search-provider';
+
+export default class NaviReportSearchProviderService extends NaviBaseSearchProviderService {
+  /**
+   * @property {Ember.Service} store
+   */
+  @service store;
+
+  /**
+   * @property {Ember.Service} user
+   */
+  @service user;
+
+  /**
+   * @property associatedComponent
+   */
+  associatedComponent = 'navi-report-search-result';
+
+  /**
+   * @method _parseQueryString
+   * @private
+   * @param {String} query
+   * @returns {Object} query object
+   */
+  _parseQueryString(query) {
+    let author = this.user.getUser().id;
+    let parsedQuery = { searchParams: null, author: author };
+
+    if (typeof query == 'string' && query) {
+      parsedQuery.searchParams = {
+        title: query,
+        request: query
+      };
+    }
+
+    return parsedQuery;
+  }
+
+  /**
+   * @method _constructSearchQuery
+   * @private
+   * @param {Object} searchParams
+   * @param {String} author
+   * @returns {Object} search query object
+   */
+  _constructSearchQuery(searchParams, author) {
```

Comment:
Don't we already do this in our fili adapter?

Reply:
No, the adapter doesn't support filtering